prow/config.yaml

```diff
@@ -1622,6 +1622,72 @@ presubmits:
       - name: docker-graph
         hostPath:
           path: /mnt/disks/ssd0/docker-graph
+  - name: pull-kubernetes-dind-e2e
+    agent: kubernetes
+    context: pull-kubernetes-dind-e2e
+    always_run: false
+    rerun_command: "/test pull-kubernetes-dind-e2e"
+    trigger: "(?m)^/test pull-kubernetes-dind-e2e,?(\\s+|$)"
+    skip_branches:
+    - release-1.6
+    - release-1.7
+    - release-1.8
+    - release-1.9
+    labels:
+      preset-service-account: true
+      preset-bazel-scratch-dir: true
+      preset-bazel-remote-cache-enabled: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bootstrap:v20180215-b2a89850e
```

Comment:
use a kubekins-e2e image instead (which has a stable version of kubetest), and you don't have to check out test-infra

Reply:
Is test-infra checked out? The build logic for dind lives in the test-infra repository now.

Reply:
yes - https://github.com/kubernetes/test-infra/blob/master/images/bootstrap/runner#L21